package.json

```diff
@@ -23,6 +23,7 @@
         "react": "^18.3.1",
         "react-day-picker": "^8.10.1",
         "react-dom": "^18.3.1",
+        "react-dotenv": "^0.1.3",
```

Comment:
Is this commonly used? it has very low downloads.

I think we should replace with `dotenv` package: https://www.npmjs.com/package/dotenv

Then just `import 'dotenv/config'` at application entrypoint file and it loads environment variables.

Reply:
Thank you for the reply. Does this package in any way hinder us from getting the job done? Are there any issues related to security or good programming practices?

Reply:
I just noticed that https://www.npmjs.com/package/react-dotenv has 7,000 weekly downloads and seems to be a wrapper for dotenv, as seen in their dependencies:
![image](https://github.com/user-attachments/assets/d9843a44-48db-40ce-b6ec-61183637b019)

I am not sure what other functionality the react-dotenv package actually offers, and if it's required in our use case. The question is rather why you would want to go with a package of 7,000 weekly downloads instead of the package it is relying on that has 40,000,000 weekly downloads? https://www.npmjs.com/package/dotenv

There might be a reason to use react-dotenv here. You are the developer implementing this, so you know better. But if you cannot give a reason, then I suggest we switch to dotenv instead of react-dotenv.

I think you can spot the downsides of using the lesser known wrapper package by using common sense. Security issues, bugs, last updated 4 years ago, etc. etc.

Reply:
The dotenv package is not meant for browsers, but rather backend applications. The react-dotenv package, however, is specifically tailored with Create-React-App (which we are using) in mind. They are not the same nor equivalent.

One of the things the react-dotenv package offers is to expose specified environment variables to the browser.

Reply:
It's only a bloated fancy wrapper consisting of 73 lines of code. Made by one guy in 2020 in a single day. 

The only thing it does seems to be injecting environment variables from .env file into the window.env global that browsers use.

react-dotenv USES dotenv.

But I'll give you this.

Reply:
Actually, I'm unresolving this conversation and requiring changes.

It took me one minute to google this. Just prefix env variables with "REACT_APP_" and they're accessible in browser.

No need to use this dumb npm package. I'll stop now here before I get banned for bad language. Hope you get the message.